addons/mail/static/src/core/common/thread_service.js

```diff
@@ -337,14 +318,14 @@ export class ThreadService {
             const previews = await this.orm.call("discuss.channel", "channel_fetch_preview", [ids]);
             for (const preview of previews) {
                 const thread = this.store.Thread.get({ model: "discuss.channel", id: preview.id });
-                const data = Object.assign(preview.last_message, {
-                    body: markup(preview.last_message.body),
-                });
-                const message = this.store.Message.insert({
-                    ...data,
-                    res_id: thread.id,
-                    model: thread.model,
-                });
+                const message = this.store.Message.insert(
+                    {
+                        ...preview.last_message,
+                        res_id: thread.id,
+                        model: thread.model,
```

Comment:
`last_message` in `channel_fetch_preview` uses `message_format` (which has `res_id` and `model`)